app/views/layouts/_footer.html.erb

```diff
@@ -1,10 +1,7 @@
 <footer class="crayons-footer print-hidden">
   <div id="footer-container" class="crayons-footer__container">
     <p class="fs-s crayons-footer__description">
-      <a href="<%= root_path %>" aria-label="<%= t("views.main.aria_home", community: community_name) %>" class="c-link c-link--branded fw-medium">
-        <%= community_name %>
-      </a>
-      – <%= Settings::Community.community_description %>
+      <%= t("views.main.footer.desc_html", community: link_to(community_name, root_path, class: "c-link c-link--branded fw-medium", "aria-label": t("views.main.aria_home", community: community_name)), desc: Settings::Community.community_description) %>
```

Comment:
This looks more convoluted, with this change, I now see nested translations.  Can you speak more to this change?

Reply:
Unfortunately, as long as you want to keep the one-line `Community — Description` style, you need to bear with this complexity. Or you might want to think about breaking up those two elements into each own line.

In English, it just looks like two pieces of text connected with an ordinary dash. But this very dash opens a can of worms.

1. some languages have different punctuation rules

[Chinese](https://w3c.github.io/clreq/#indication_punctuation_marks), [Japanese](https://www.w3.org/TR/jlreq/#unbreakable_character_sequences), and [Korean](https://w3c.github.io/clreq/#indication_punctuation_marks) always use dashes in twos, with no surrounding spacing. So you have to change like this:
```
Community——Description
```

2. some languages don't use dash

[Tibetan](https://www.w3.org/International/tlreq/#head_marks) has a long tradition of literacy with rich paragraph-level punctuation. They have native headline style that is used like this (if I understand it correctly):
```
༄༅།Community། །Description།
```

3. some languages need to adjust elsewhere because of this dash

The dash in the middle can be a big headache when you support RTL languages. [MDN already conveniently has a good summary on this matter](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi) instead of me. In short, if you have such a run of text:
```
DEV.TO - 42 או קבלו הקהילה
```
the default HTML rendering will be a mess in RTL environment:
> <p dir="rtl">DEV.TO - 42 או קבלו הקהילה</p>

instead you have to add format controls like:
```
<bdi>DEV.TO</bdi> - 42 או קבלו הקהילה
```
to get expected output (sadly GitHub strips off those BiDi tags, so I show you in image):
![image](https://user-images.githubusercontent.com/84892012/146817246-0def7c9f-d6c0-42fb-8993-f62d23d006c9.png)

Reply:
Thank you for the greater context.  Your proposed change does address those complexities.